redfish-core/lib/processor.hpp

```diff
@@ -1358,10 +1688,63 @@ inline void requestRoutesProcessorCollection(App& app)
         asyncResp->res.jsonValue["@odata.id"] =
             "/redfish/v1/Systems/system/Processors";
 
-        collection_util::getCollectionMembers(
-            asyncResp,
-            boost::urls::url("/redfish/v1/Systems/system/Processors"),
-            processorInterfaces, "/xyz/openbmc_project/inventory");
+        crow::connections::systemBus->async_method_call(
+            [asyncResp](const boost::system::error_code ec,
+                        const std::vector<std::string>& objects) {
+            if (ec)
+            {
+                BMCWEB_LOG_DEBUG("DBUS response error");
```

Comment:
Message here should be using BMCWEB_LOG_ERROR() since we are returning an error.

It should also include ec.value() to help with debugging.

Reply:
I see the same BMCWEB_LOG_DEBUG used in other places too. So will address all of them together. 

Will address this also as part of new commit (JIRA number will be attached)